app/views/registrations/new.haml

```diff
@@ -0,0 +1,44 @@
+#registration
+  .container-fluid
+    .row-fluid
+      .span10.offset1
+        .span7.hidden-phone
+          %h1#create-something-text
+            = t('.hey_make').html_safe
+          %h3#diaspora-hearts
+            = t('.diaspora')
+        .span5
+          .text-center
+            %h2#pod-name
+              = AppConfig.settings.pod_name
+            = form_for(resource, url: registration_path(resource_name), html: {class: "form-horizontal block-form", autocomplete: "off"}) do |f|
+              %fieldset
+                %label.control-label{for: "user_email"}
+                  = t('.email')
+                %i.entypo.mail
+                = f.email_field :email, class: "input-block-level form-control", placeholder: t('.email'), title: t('registrations.new.enter_email'), required: true
+
+                %label.control-label{for: "user_username"}
+                  = t('.username')
+                %i.entypo.user
+                = f.text_field :username, class: "input-block-level form-control", placeholder: t('.username'), title: t('registrations.new.enter_username'), required: true, pattern: "[A-Za-z0-9_]+"
+
+                %label.control-label{for: "user_password"}
+                  = t('.password')
+                %i.entypo.lock
+                = f.password_field :password, class: "input-block-level form-control", placeholder: t('.password'), title: t('registrations.new.enter_password'), required: true, pattern: "......+"
+
+                %label.control-label{for: "user_password_confirmation"}
+                  = t('.password_confirmation')
+                %i.entypo.lock
+                = f.password_field :password_confirmation, class: "input-block-level form-control", placeholder: t('.password_confirmation'), title: t('registrations.new.enter_password_again'), required: true, pattern: "......+"
+
+                - if AppConfig.settings.captcha.enable?
+                  = show_simple_captcha :object => 'user', :code_type => 'numeric'
+
+                = invite_hidden_tag(invite)
+
+              - if AppConfig.settings.terms.enable?
+                %p#terms.text-center
+                  = t('.terms', terms_link: link_to(t('.terms_link'), terms_path, target: "_blank")).html_safe
+              = f.submit t('.sign_up'), class: "btn", data: {disable_with: t('.submitting')}
```

Comment:
Hm, is `devise.shared.links.sign_up` used elsewhere? Because it is translated by "Inscription" in French, which is a name. I'd like a verb here.